db/models/articles.model.js

```diff
@@ -1,26 +1,48 @@
 const db = require('../connection');
 
-function idCheck(rows) {
-    if(rows.length !== 0){
+function idCheck(id, rows) {
+    const values = [id]
+   if (rows.length === 0) {
+    return db.query(`SELECT * FROM articles WHERE article_id = $1;`, values).then(({rows}) => {
         return rows
-    }
-    else return {Error : "ID Does Not Exist"}
+    })
+    .then((rows) => {
+        if(rows.length !== 0){
+            return Promise.reject('no comments')
+        }
+        else {
+           return Promise.reject('wrong id')
```

Comment:
We should be able to rely on sql to throw an error that we can catch with an error code of I think '22P02' for a bad ID. So no need for a custom error in all circumstances. Also the standard format for a promise.reject is something like this:

return Promise.reject({
       status: 404
       message: 'bad request'
});

So we can then let our error handlers pick this up and have the status and message be dynamic.